js/src/api/util/address.js

```diff
@@ -0,0 +1 @@
+../../abi/util/address.js
```

Comment:
@ngotchac Maybe i missed something, but then `abi` would depend on `api`, which we try to prevent.